packages/neon/neon/lib/src/utils/request_manager.dart

```diff
@@ -68,6 +68,7 @@ class RequestManager {
         disableTimeout,
       );
 
+  @experimental
```

Comment:
Why experimental? I think this is properly working?

Reply:
I want webdav to also make use of DynamiteRawResponses once we implement them in a different more general way.
This would allow us to hopefully deserialize them on a separate isolate (as is the hope for the raw response).

I'll remove the annotation for now as we can just annotate it with deprecated once we have it in place.